README.md

```diff
@@ -4,7 +4,7 @@
 
 * This is application that is designed for busy beauty salons with diverse customer bases that want to personalise service.<br>
   Example usages:
-  * Track customer profiles like their skin type or hair type so that appropriate products can be recommended for return customers
+  * Track customer profiles like their skinType type or hairType type so that appropriate products can be recommended for return customers
```

Comment:
I don't think there's a need to replace the skin and hair to "skinType" and "hairType" respectively as skinType and hairType is more of the naming of the attribute in the Customer class?

Reply:
edited